### PR TITLE
cleanup logic

### PR DESCRIPTION
**Pull request by liudmylailchenko**
Added logic for cleaning dumps files

### CODE REVIEW

File: src/actions.py [hunk 1]
```diff
@@ -421,3 +423,26 @@ def add_note(args: list[str], notes_book: "NoteBook") -> None:
...
+    """
+    config = args[0]
+
+    files_to_delete = {
+        "all": [ADDRESS_BOOK_FILE, NOTES_FILE],
+        "address-book": [ADDRESS_BOOK_FILE],
+        "notes": [NOTES_FILE],
+    }
+    if files_to_delete.get(config) is None:
+        raise NotFoundWarning("Invalid config value. Choose from 'all', 'address-book', or 'notes'.")
```
Comment (vp-mx) [inline src/actions.py]: please change slightly message
https://github.com/vp-mx/pocketpal/blob/af1dd1e1af80154169eacbd073cd1cfed758db8c/src/error_handlers.py#L44

e.g. Choose from 'all', 'address-book', or 'notes'.  Files for {config}

as prefix not found will be appended automatically

**Review by vp-mx — APPROVED** [review]

File: src/actions.py (hunk 1 above)
Comment (liudmyla-ilchenko) [reply src/actions.py]: Fixed

Comment (liudmylailchenko) [reply src/actions.py]: Fixed

**Comment by AnnetaDe** [comment]
мейн не свіжий

**Pull request closed without merging**